karma.conf.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
```

Comment:
Please clarify. Is this all the JavaScript we'd need to run isparta without karma / karma-coverage?

```js
var instrumenter = new isparta.Instrumenter(assign({
  embedSource: true,
  noAutoWrap: true
}, options));
```

**If so, that would be easy to do without the overhead and context switching of karma, and we should just use a little JS instead of a heavy-handed tool.**

**The overhead and context switching of karma may not be worth the cost of perfect-looking HTML coverage reports.**


## Why not Karma?

### We already have the mostly good-enough case

What we really need from a coverage automation tool is pass/fail and then then ability to figure out why it failed. That second part is the only awkward thing about the coverage reports without isparta.

**This problem *will* get fixed upstream in standard tools**


### The cost of Karma may be too high

**This repo will be the base of MANY other projects.** I don't think that getting perfect coverage reporting *right now* is worth the cost of introducing & **eventually extracting** Karma from every repo that inherits the config from this repo.

This last point may be true just for my personal needs, but consider that many other people may also use this repo, especially if I make it the basis for a scaffolding tool (a logical next step).